query/src/sql/sql_common.rs

```diff
@@ -41,32 +43,67 @@ impl SQLCommon {
             //custom types for databend
             // Custom(ObjectName([Ident { value: "uint8", quote_style: None }])
             SQLDataType::Custom(obj) if !obj.0.is_empty() => {
-                match obj.0[0].value.to_uppercase().as_str() {
-                    "UINT8" => Ok(DataType::UInt8),
-                    "UINT16" => Ok(DataType::UInt16),
-                    "UINT32" => Ok(DataType::UInt32),
-                    "UINT64" => Ok(DataType::UInt64),
+                let value = obj.0[0].value.to_uppercase();
```

Comment:
Is it the right place to parse enum definition?

Reply:
No, I'm afraid it's not the right place.

We should modify the codes in sqlparser, because currently it only accept a single token word as DataType which will not parse `Enum('a' = 0, 'b' = 1, 'c' = 2, 'd' = 4)`.